### DISCUSSION TITLE
Observable and article layout: is it possible to use it?

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#4216] [Q&A]
Hi,
I have the below sample code.

~~~~
```{ojs}
//| echo: false
//| column: page
data = FileAttachment("input.csv").csv({ typed: true })
Inputs.table(data)
```
~~~~

I would like to apply [Page Column](https://quarto.org/docs/authoring/article-layout.html#page-column), but it seems not work for observable.

Thank you

**Comment by cscheid** [comment]
Yes, that's a bug. Will open an issue to track.
